dev.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM weblate/weblate:edge
+
+##########
+# WARNING!
+# This dockerfile is meant to be used in the development process
+# and WILL perform very poorly in production.
+#
+# For production-ready dockerfile see:
+# https://github.com/WeblateOrg/docker/tree/docker
+#########
+
+# TODO: put some new dependencies here
```

Comment:
You can put new development dependencies here

@nijel I should probably add `pip install -r requirements.txt` here if it installs packages globally, may be something else?

Reply:
`pip install` should IMHO work fine here.